SearchAPI/Baseline/Stack.py

```diff
@@ -92,13 +92,16 @@ def get_stack_params(reference, product_type=None):
         stack_params['platform'] = get_platform(reference)
         stack_params['beamMode'] = reference_results[0]['beamMode']
         stack_params['flightDirection'] = reference_results[0]['flightDirection']
-        stack_params['lookDirection'] = reference_results[0]['lookDirection']
+        if stack_params['processingLevel'] != 'BURST':
+            stack_params['lookDirection'] = reference_results[0]['lookDirection']
         stack_params['relativeorbit'] = reference_results[0]['relativeOrbit'] # path
         stack_params['polarization'] = reference_results[0]['polarization']
-        if stack_params['polarization'] in ['HH', 'HH+HV']:
-            stack_params['polarization'] = 'HH,HH+HV'
-        elif stack_params['polarization'] in ['VV', 'VV+VH']:
-            stack_params['polarization'] = 'VV,VV+VH'
+        
+        if stack_params['processingLevel'] != 'BURST':
```

Comment:
Not sure this is right: if the stack is made of bursts, the polarization must match _precisely_, rather than allowing the more generous dual-pol matching that SLC's allow. My understanding is that a dual-pol SLC (HH+HV, for example) is what it is not due to each burst being dual-pol, but instead due to having bursts of each type. Grabbing all the bursts from an HH+HV SLC would yield a bunch of HH bursts and a bunch of HV bursts. I think.

Anyhow, I think that means all this can actually go away and the reference burst pol can just be used directly, as-is, for the stack params.

Reply:
I believe that is the current behavior, the polarization is assign from the reference scene by default in the line just above the if block. I grouped them closer together and also moved an earlier `if not burst` block into this one as well.